packages/shared/src/components/post/ShareYouTubeContent.tsx

```diff
@@ -18,8 +19,13 @@ function ShareYouTubeContent({
   post,
   onReadArticle,
 }: ShareYouTubeContentProps): ReactElement {
-  if (!post.sharedPost) {
-    return <></>;
+  if (!post.sharedPost?.id) {
+    return (
+      <>
+        <ElementPlaceholder className="mt-6 h-6 w-2/4 rounded-10" />
+        <ElementPlaceholder className="my-5 w-full rounded-16 pt-[56.25%]" />
+      </>
+    );
```

Comment:
More of a note (I personally think its fine), if any error in client side fetching happens the placeholder would stay forever?

Reply:
+ think most apps also do that

Reply:
Uhm no I think we redirect on error actually. 
Just need to make minor changes in the squad directory I see.

I did check but missed one screensize 😅

Reply:
From static props, but I mean client side fetch in useQuery?

Reply:
Ill double check that 👀